### PR TITLE
Improve description visuals

### PR DESCRIPTION
**Pull request by olu144**
Screenshot 2020-07-09 at 9 21 37 PM - Display 2

put comments in jumbotron for improved visuals

### CODE REVIEW

**Comment by danieldortega77** [comment]
I'm going to wait until the other pages are merged with master before reviewing; it's hard to pin down what exactly is different from the other PRs just from looking at the diffs

**Comment by chrismohrii** [comment]
> I'm going to wait until the other pages are merged with master before reviewing; it's hard to pin down what exactly is different from the other PRs just from looking at the diffs

+1

File: capstone/src/main/webapp/document_analyzer.html
Comment (danieldortega77) [inline capstone/src/main/webapp/document_analyzer.html]: nit: Let's add a newline

File: capstone/src/main/webapp/image_analyzer.html
Comment (danieldortega77) [inline capstone/src/main/webapp/image_analyzer.html:42]: Depending on API/implementation details, this might have to change later on. For now, this is good!

**Review by danieldortega77 — APPROVED** [review]

File: capstone/src/main/webapp/text_analyzer.html
Comment (chrismohrii) [inline capstone/src/main/webapp/text_analyzer.html:44]: nit: Are we sure this detail should be included? I would assume the user would know this is inferred everywhere they can write something

**Review by chrismohrii — APPROVED** [review]

File: capstone/src/main/webapp/document_analyzer.html
Comment (olu144) [reply capstone/src/main/webapp/document_analyzer.html]: addressed in "add newline"

File: capstone/src/main/webapp/image_analyzer.html
Comment (olu144) [reply capstone/src/main/webapp/image_analyzer.html:42]: Noted! I'll keep that in mind

**Review by dborkan — APPROVED** [review]
👍 Code looks good.  Also for PRs like this it might also be good to include an image in the PR description so other reviewers can see the new layout.

File: capstone/src/main/webapp/document_analyzer.html
Comment (dslucas) [inline capstone/src/main/webapp/document_analyzer.html:40]:
```suggestion
            <h1>Document Analysis</h1>
```

**Review by dslucas — APPROVED** [review]

**Pull request merged**
